etl/steps/data/garden/energy/2023-12-12/primary_energy_consumption.meta.yml

```diff
@@ -1,37 +1,38 @@
-# TODO: This file is a duplicate of the previous step. It is not yet used in the dag and should be updated soon.
+definitions:
+  common:
+    processing_level: major
+    presentation:
+      topic_tags:
+        - Energy
+    description_processing: |
+      - Primary energy consumption data was compiled based on two key data sources: [Energy Institute (EI) Statistical Review of World Energy](https://www.energyinst.org/statistical-review), and [International energy data from the U.S. Energy Information Administration (EIA)](https://www.eia.gov/international/data/world/total-energy/more-total-energy-data). EI provides the longest and most up-to-date time-series of primary energy. However, it does not provide data for all countries. We have therefore supplemented this dataset with energy data from the EIA. Where EI provides data for a given country, this data is adopted; for countries where this data is missing, we rely on EIA energy figures.
+      - Per capita figures have been calculated using a population dataset that is based on [different sources](https://ourworldindata.org/population-sources).
+      - To calculate energy per unit of GDP, we divide by total real GDP figures from [the Maddison Project Database, version 2020](https://www.rug.nl/ggdc/historicaldevelopment/maddison/releases/maddison-project-database-2020).
 
 dataset:
-  title: Primary energy consumption (EI & EIA, 2023)
-  description: |
-    Primary energy consumption data was compiled by Our World in Data based on two key data sources:
-    1. [Energy Institute (EI) Statistical Review of World Energy](https://www.energyinst.org/statistical-review).
-    2. [International energy data from the U.S. Energy Information Administration (EIA)](https://www.eia.gov/international/data/world/total-energy/more-total-energy-data).
+  update_period_days: 365
 
-    EI provides the longest and most up-to-date time-series of primary energy. However, it does not provide data for all countries. We have therefore supplemented this dataset with energy data from the EIA. Where EI provides data for a given country, this data is adopted; for countries where this data is missing, we rely on EIA energy figures.
-
-    Per capita figures have been calculated using a population dataset that is built and maintained by Our World in Data, based on [different sources](https://ourworldindata.org/population-sources).
-
-    To calculate energy per unit of GDP, we use total real GDP figures from [the Maddison Project Database, version 2020](https://www.rug.nl/ggdc/historicaldevelopment/maddison/releases/maddison-project-database-2020).
 tables:
   primary_energy_consumption:
+    title: Primary energy consumption
```

Comment:
you don't seem to have any `description_short/ description_key` in your metadata files that otherwise exists in your charts and is useful - e.g.,` [Primary energy](#dod:primaryenergy) consumption is measured in terawatt-hours (TWh)`. You can of course just leave it as it is and edit the charts directly or you could actually use the fields we now have to add these. I won't go over each one and this is just an example.